pkg/manager/impl/validation/validation.go

```diff
@@ -268,3 +276,21 @@ func ValidateOutputData(outputData *core.LiteralMap, maxSizeInBytes int64) error
 	}
 	return errors.NewFlyteAdminErrorf(codes.ResourceExhausted, "Output data size exceeds platform configured threshold (%+v > %v)", outputSizeInBytes, maxSizeInBytes)
 }
+
+func ValidateDatetime(literal *core.Literal) error {
+	if literal == nil {
+		return nil
+	}
```

Comment:
does this nil check make sense?

the fixedInput should not be nil as far as i understand.
This default behaviour will not protect us from future refactorings

Reply:
Do you mean that we should fail if literal == nil? Or that it will never happen?

Reply:
I think so and if necessary do refactorings on the places of usage to accomodate for the nil usecase

Reply:
As mentioned via private message, I changed the check to throw an error instead. Just to be on the safest side.